README.md

```diff
@@ -23,16 +23,16 @@ you run the first build.
 
 [install]: https://bazel.build/versions/master/docs/install.html
 
-Also add `ibazel` to your `$PATH`:
+Also add `bazel` to your `$PATH`:
```

Comment:
your commit message should include *why*

ibazel is the watch mode and installed by npm, so the original is correct here, not sure what you saw that suggested it should change...